themes/digital.gov/layouts/partials/core/guides/guide-prev-next.html

```diff
@@ -1,48 +1,34 @@
-{{- $guide_data := index $.Site.Data.guidenav -}}
+{{- $guideData := index $.Site.Data.guidenav -}}
 {{ if .Params.guide }}
-  {{- $guide_data = index $.Site.Data.guidenav (.Params.guide) -}}
+  {{- $guideData = index $.Site.Data.guidenav (.Params.guide) -}}
 {{ end }}
 
-{{- if $guide_data.showNextPrevious -}}
-  {{- $current_page := . -}}
-  {{- $current_url := $current_page.Permalink -}}
+{{- if $guideData.showNextPrevious -}}
+  {{- $currentPage := . -}}
+  {{- $currentURL := $currentPage.Permalink -}}
 
-  {{- $guide_files := (where .Site.Pages ".Params.guide" .Params.guide) -}}
+  {{- $prevItem := 0 -}}
+  {{- $pointItem := 0 -}}
```

Comment:
We use `$prevItem` and `$prevPage` which makes `$pointerItem` and `$pointerPage` a bit confusing at first. I would suggest updating it to use `$nextItem` and `$nextPage` to improve readability.